StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/EmbeddedPaymentMethodsViewTests.swift

```diff
@@ -50,13 +50,13 @@ final class EmbeddedPaymentMethodsViewTests: XCTestCase {
 
         // Simulate tapping Cash App and verify delegate is called
         embeddedView.didTap(rowButton: embeddedView.getRowButton(accessibilityIdentifier: "Cash App Pay"))
-        XCTAssertEqual(mockDelegate.calls, [.didUpdateSelection, .didUpdateHeight, .didTapPaymentMethodRow])
+        XCTAssertEqual(mockDelegate.calls, [.didUpdateHeight, .didUpdateSelection, .didTapPaymentMethodRow])
```

Comment:
Now that we update the mandate before notifying delegate of the payment option changing this ordering changes. This seems reasonable as it's pretty much simultaneous.

Reply:
We use `layoutSubviews` to auto listen for these height changes which triggers the didUpdateHeight, I think I prefer keeping it there than trying to keep track of height changes on our own to control the ordering, so I'm proposing we call didUpdateHeight before didUpdateSelection in this case.